### ISSUE TITLE
Building error for latest v4.13.0

### ISSUE THREAD

**Issue by dhammel** [vladimiry/ElectronMail#463]
Hello,

normally I do not get errors building this project, but this one I can't seem to resolve.

I'm not sure exactly what the error is caused by, I have a link to pastebin for the whole log:

https://pastebin.com/kP7SFPLP

I am building on linux, Kubuntu 21.10. I have all needed dependencies installed, plus nvm, node version 16.13.0, npm version 7.24.1 (at first, tried the latest 8.1.3), and yarn version 1.22.17.

yarn --pure-lockfile seems to be successful, during the yarn run app:dist, it errors out with "Command failed with exit code 1" at point which appears to be duing the build of Proton WebClient.

I normally build after checkout out a recent release tag, but this one I did on default branch after it first started error out.

Any clue what seems to be occuring? Nothing about my environment changed, other than upgrading node & npm as needed according to readme.

Thank you

**Comment by vladimiry** [comment]
> it errors out with "Command failed with exit code 1" at point which appears to be duing the build of Proton WebClient.

Some options (try it one by one from the beginning):
- Remove the `{printStdOut: false}` part at this line https://github.com/vladimiry/ElectronMail/blob/0133434c1950832e76c7487df6ec31f09bd1a0c4/scripts/prepare-webclient/webclients.ts#L282 So you get command output posted to console. If the output is still truncated just run the last `yarn install` directly being located in the `./output/git/WebClients` folder adn see if it completes without errors.
- Replace the `if (process.env.CI) {` with `if (true)` on https://github.com/vladimiry/ElectronMail/blob/0133434c1950832e76c7487df6ec31f09bd1a0c4/scripts/prepare-webclient/webclients.ts#L259.
- Try to build the @protonmail web clients directly as described in https://github.com/ProtonMail/WebClients. You will have to find a way to hardcode the proper API Entry point though and then put the built client to the folders structure the app expected it to be located in.
- Try the `wip` branch.
- Clone the project and let the GitHub Ations CI handle the stuff.

**Comment by dhammel** [comment]
@vladimiry 

Building from the WIP branch was successful.

> Remove the {printStdOut: false} part at this line 

All this did was cause a lot of extra logging, as you mentioned it would but did not resolve anything at all, same error.

> Replace the if (process.env.CI) { with if (true) on 

This resulted in an error within the first 10 seconds of running yarn run app:dstt that stated that an expression was required, though I kept it the same format as the rest of the file, and edited it exactly as you mentioned.

> Try to build the @ProtonMail web clients directly as described in https://github.com/ProtonMail/WebClients. You will have to find a way to hardcode the proper API Entry point though and then put the built client to the folders structure the app expected it to be located in.

I did not try this method, as I managed to build from WIP branch successfully, maybe i'll try later if neccesarry or if you would like to know the results for some reason or another.

> Clone the project and let the GitHub Ations CI handle the stuff for you.

I am not familiar with Github Actions CI, though I would prefer to do everything locally if at all possible.

Would it be discouraged to use a build from the WIP branch for Production use? At least until you merge changes, and release a new build.

Thank you for your help
